hystrix/src/main/java/feign/hystrix/HystrixInvocationHandler.java

```diff
@@ -38,11 +39,23 @@
   private final Target<?> target;
   private final Map<Method, MethodHandler> dispatch;
   private final Object fallback; // Nullable
+  private final Map<Method, Method> toFallback;
 
   HystrixInvocationHandler(Target<?> target, Map<Method, MethodHandler> dispatch, Object fallback) {
     this.target = checkNotNull(target, "target");
     this.dispatch = checkNotNull(dispatch, "dispatch");
     this.fallback = fallback;
+    this.toFallback = toFallbackMethod(dispatch);
```

Comment:
I do have tried this way at first. But it doesn't take effect. The method injected in InvocationHandler is not the same as method in dispatch, though they are equals. So i bypass this problem by saving a map.